src/_guides/language/language-tour.md

```diff
@@ -3536,8 +3536,11 @@ Static variables aren’t initialized until they’re used.
 
 #### Static methods
 
-Static methods (class methods) do not operate on an instance, and thus
-do not have access to `this`. For example:
+Static methods (class methods) don't operate on an instance, and thus
+don't have access to `this`.
```

Comment:
You could also comment on the fact that they can't reference generic parameters to the class (or not).